services.yaml

```diff
@@ -909,49 +926,197 @@ paths:
           $ref: "#/definitions/UnauthorizedError"
+  /services/{service_id}/folders:
+    parameters:
+      - in: path
+        name: service_id
+        description: Environment to use
+        type: string
+        required: true
+    get:
+      tags:
+        - View Folders
+        - VFs
+      summary: "List View Folders"
+      description: "List View Folders"
+      responses:
+        200:
+          description: List of VF names
+          schema:
+            properties:
+              vf_names:
+                type: "array"
+                items:
+                  type: "object"
+                  properties:
+                    name:
+                      $ref: "#/definitions/Folder"
+        401:
+          $ref: "#/definitions/UnauthorizedError"
+  /services/{service_id}/folders/{folder_id}/views:
+    parameters:
+      - in: path
+        name: service_id
+        description: Environment to use
+        type: string
+        required: true
+      - in: path
+        name: folder_id
+        description: View folder to use
+        type: string
+        required: true
+    post:
+      tags:
+        - View Folders
+        - VFs
+      summary: "Add view to selected folder"
+      description: "Add view to selected folder"
+      parameters:
+        - in: body
+          description: View ID (string)
+          schema:
+            name: view_id
+            type: String
+            example: P12
+          required: true
+      responses:
+        201:
+          description: CREATED
+        404:
+          description: NOT FOUND
+        401:
+          $ref: "#/definitions/UnauthorizedError"
   /services/{service_id}/views:
+    parameters:
+        - in: path
+          name: service_id
+          description: Environment to use
+          type: string
+          required: true
     get:
       tags:
         - Views
-      summary: "List views"
-      description: "List views"
+      summary: "List Views"
+      description: "List Views"
+      responses:
+        200:
+          description: List of views including basic info about each view.
+          schema:
+            properties:
+              views:
+                type: "array"
+                items:
+                  $ref: "#/definitions/ViewBasic"
+        401:
+          $ref: "#/definitions/UnauthorizedError"
+    post:
+      tags:
+        - Views
+      summary: "Create View"
+      description: "Create View"
       parameters:
-          - in: path
-            name: service_id
-            description: Environment to use
-            type: string
-            required: true
-          - in: query
-            name: name
-            description: Name of view(s) to fetch
-            type: string
+        - in: body
+          name: view
+          description: Single view properties
+          schema:
+            properties:
+              $ref: "#/definitions/ViewFull"
+      responses:
+        201:
+          description: CREATED
```

Comment:
here, should return the Location of the new Pxxxx

Reply:
You mean the url? or just the id?